app/src/main/java/de/langerhans/odintools/tools/SettingsRepo.kt

```diff
@@ -0,0 +1,110 @@
+package de.langerhans.odintools.tools
+
+import de.langerhans.odintools.BuildConfig
+import javax.inject.Inject
+
+class SettingsRepo @Inject constructor(
+    private val executor: ShellExecutor,
+) {
+
+    init {
+        println("Settings init")
```

Comment:
For logging use `Log.d()` or other levels. Maybe we should think about adding a logging library like Timber to simplify this.

Reply:
Oops! Slipped from my sight. Good catch!